code/modules/mining/parachute_and_ropes.dm

```diff
@@ -0,0 +1,49 @@
+/obj/item/parachute
```

Comment:
you are aware this is useable on all multiz? it should have a description denoting it as such

Reply:
yep